src/sagemaker/jumpstart/curated_hub/utils.py

```diff
@@ -0,0 +1,41 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Utilities to interact with Hub."""
+from typing import Any, Dict
+import boto3
+from sagemaker.session import Session
+from sagemaker.jumpstart.types import HubDescription, HubContentType, HubContentDescription
+
+
+def describe(hub_name: str, region: str) -> HubDescription:
+    """Returns descriptive information about the Hub."""
+
+    sagemaker_session = Session(boto3.Session(region_name=region))
+    hub_description = sagemaker_session.describe_hub(hub_name=hub_name)
+    return HubDescription(hub_description)
+
+
+def describe_model(
+    hub_name: str, region: str, model_name: str, model_version: str = "*"
+) -> HubContentDescription:
+    """Returns descriptive information about the Hub model."""
+
+    sagemaker_session = Session(boto3.Session(region_name=region))
```

Comment:
For both fns in this file, I think we should 1/ accept a custom SM Session and 2/ default to the `DEFAULT_JUMPSTART_SAGEMAKER_SESSION`

Reply:
Reason behind not using the `DEFAULT_JUMPSTART_SAGEMAKER_SESSION` was because `Hub` is technically not just for JS.